interfaces/IBF-dashboard/src/assets/i18n/en.json

```diff
@@ -41,49 +41,91 @@
-      "welcome": "To your right is the map of your country. You can turn other layers on and off. There are currently <strong>no triggers</strong> activated."
-    },
-    "active-event-active-trigger": {
-      "welcome": "A trigger warning for <strong>{{ disasterTypeLabel }}</strong> was activated on <strong>{{ startDate }}</strong>. It is now <strong>{{ leadTime }} {{ timeUnit }}(s)</strong> away."
-    },
-    "active-event-no-trigger": {
-      "welcome": "There is <strong>no active trigger</strong> any more, as can be seen in the map, which always reflects the latest forecast. However, you can still see here the EAP-actions that relate to the event that started on <strong>{{ startDate }}</strong> and ended on <strong>{{ endDate }}</strong>. This event stays visible here for 7 days after the end-date."
-    },
-    "active-event": {
-      "instruction": "Please select the area on the map to monitor, select and send the pre-planned anticipatory actions executed in each area and eventually close the event. These actions are only available on the main admin-level.",
-      "place-name": "{{ adminAreaLabel }}: <strong>{{ placeName }}</strong>",
-      "exposed": "Exposed Population: <strong>{{ exposedNumber }}</strong>",
-      "submit-button-label": "Save",
-      "close-event-button": "Close Alert",
-      "update-success": "EAP action(s) updated in database.",
-      "update-failure": "Failed to update EAP action(s) in database.",
-      "prompt-button-label": "OK",
-      "close-event-popup": {
-        "message": "Are you sure? You are about to close the alert in {{ placeCodeName }}. This action can not be undone.",
-        "confirm": "Close Alert",
-        "cancel": "Cancel",
-        "failure": "Failed to close alert."
+    "floods": {
+      "about-button-label": "About Trigger",
+      "warn-label": {
+        "message": "Hello {{ name }}. The information in this portal is based on the model last run on <strong>{{lastModelRunDate}}</strong>."
+      },
+      "no-event-no-trigger": {
+        "welcome": "To your right is the map of your country. You can turn other layers on and off. There are currently <strong>no triggers</strong> activated."
+      },
+      "no-event-active-trigger": {
+        "welcome": "A trigger warning for a lead time of {{ activeLeadTime }} days was first activated on <strong>{{ startDate }}</strong> and has remained in trigger mode ever since. The event is now predicted to arrive in <strong>{{ leadTime }} {{ timeUnit }}(s)</strong>."
+      },
+      "active-event-active-trigger": {
+        "welcome": "A trigger warning for a lead time of {{ activeLeadTime }} days was first activated on <strong>{{ startDate }}</strong> and has remained in trigger mode ever since. The event is now predicted to arrive in <strong>{{ leadTime }} {{ timeUnit }}</strong>, but has been continually flagged to arrive in {{ leadTime }} {{ timeUnit }} for the last <strong>{{ timeOngoing }} {{ timeUnit }}s</strong>."
```

Comment:
```suggestion
        "welcome": "A trigger warning for a lead time of {{ activeLeadTime }} days was first activated on <strong>{{ startDate }}</strong> and has remained in trigger mode ever since. The event is now predicted to arrive in <strong>{{ leadTime }} {{ timeUnit }}(s)</strong>, but has been continually flagged to arrive in {{ leadTime }} {{ timeUnit }}(s) for the last <strong>{{ timeOngoing }} {{ timeUnit }}s</strong>."
```

![image](https://user-images.githubusercontent.com/63000678/153567743-38c42dba-d5df-4aed-bfa2-be5412fb37e6.png)